src/CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ/MassTransitClientExtensions.cs

```diff
@@ -0,0 +1,78 @@
+﻿using MassTransit;
+using MassTransit.Monitoring;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using System.Reflection;
+
+namespace CommunityToolkit.Aspire.Client.MassTransit.RabbitMQ;
```

Comment:
The namespace for a client integration extension should be `Microsoft.Extensions.Hosting`